packages/flutter_tools/test/general.shard/version_test.dart

```diff
@@ -484,7 +492,7 @@ void main() {
   testUsingContext('determine does not call fetch --tags', () {
     final MockProcessUtils processUtils = MockProcessUtils();
     when(processUtils.runSync(
-      <String>['git', 'fetch', 'https://github.com/flutter/flutter.git', '--tags'],
+      <String>['git', 'fetch', kFlutterGit, '--tags'],
```

Comment:
I don't think you should re-use flutterGit here, since this is now secretly looking up the value on the Platform.

The test is easier to read with a constant